config/coaddBase.py

```diff
@@ -0,0 +1,9 @@
+# Configs shared between makeWarp and assembleCoadd
+config.matchingKernelSize = 29
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)